src/launcher.tsx

```diff
@@ -125,8 +169,36 @@ function LauncherBody(props: {
               item.label.toLowerCase().indexOf(query.toLowerCase()) !== -1
           )
           .map(item => <TypeCard item={item} trans={trans} />)
-    },
-    {
+    });
+  }
+  if (showStarred) {
+    builtinSections.push({
+      className: 'jp-Launcher-openByKernel',
+      title: trans.__('Starred'),
+      icon: starIcon,
+      id: 'starred',
+      rank: 2,
+      render: () =>
+        starred.length > 0 ? (
+          <KernelTable
+            items={starred}
+            commands={props.commands}
+            showSearchBox={!searchAll}
+            showWidgetType={true}
+            query={query}
+            settings={props.settings}
+            trans={trans}
+            onClick={item => item.execute()}
+            favouritesChanged={props.favouritesChanged}
+            lastUsedChanged={props.lastUsedChanged}
+          />
+        ) : (
+          'No starred items'
```

Comment:
Unrelated to this PR, but while we are at it:

```suggestion
          trans.__('No starred items')
```